sqlite-cloud/quick-start-fastapi-sqlalchemy.mdx

```diff
@@ -37,20 +37,10 @@ pip install "fastapi[standard]" sqlalchemy sqlalchemy-sqlitecloud
 
   - Do NOT remove the quotes around the FastAPI package.
   - `sqlalchemy-sqlitecloud` includes `sqlitecloud`, so no need to install the latter separately.
-  - Update `.venv/lib/python{version}/site-packages/sqlitecloud/__init__.py`:
-
-```py
-from .dbapi2 import *
-
-# from .dbapi2 import (
-#     PARSE_COLNAMES,
-#     ...
-# )
-```
```

Comment:
Just a note: change lines in the `venv` files is not correct since the folder is managed by `pip`. Changes applied there are highly volatile. :+1:

Reply:
Yes, this was a hack/ workaround! Removed now.